.riot/requirements/1aa652f.txt

```diff
@@ -5,44 +5,46 @@
 #    pip-compile --allow-unsafe --no-annotate .riot/requirements/1aa652f.in
 #
 aiobotocore==2.3.1
-aiohttp==3.9.5
-aioitertools==0.11.0
+aiohappyeyeballs==2.4.3
+aiohttp==3.10.10
+aioitertools==0.12.0
 aiosignal==1.3.1
-attrs==23.2.0
+attrs==24.2.0
 botocore==1.24.21
-certifi==2024.7.4
-charset-normalizer==3.3.2
-coverage[toml]==7.6.0
-elastic-transport==8.13.1
-elasticsearch==8.14.0
+certifi==2024.8.30
+charset-normalizer==3.4.0
+coverage[toml]==7.6.4
+elastic-transport==8.15.1
+elasticsearch==8.15.1
 events==0.5
-frozenlist==1.4.1
+frozenlist==1.5.0
 gevent==22.10.2
```

Comment:
Doesn't look like `gevent` is being updated properly. Is it pinned?

Reply:
Maybe these instances are where it is pinned for these runtimes? https://github.com/DataDog/dd-trace-py/blob/b7c97b59a2b8fa471a8fe3dc30cdf3309bbc88ef/riotfile.py#L435

Reply:
Yes, we still want to test those other versions of `gevent` that are pinned, upgrading `latests` updates the lockfiles for those riot hashes that do use `gevent latest`.

Reply:
Great thanks for confirming! I didn't review the entire PR before commenting :)